jerry-core/jrt/jrt.h

```diff
@@ -61,7 +60,7 @@
 #define JERRY_STATIC_ASSERT_GLUE(a, b, c) JERRY_STATIC_ASSERT_GLUE_ (a, b, c)
 #define JERRY_STATIC_ASSERT(x, msg) \
   typedef char JERRY_STATIC_ASSERT_GLUE (static_assertion_failed_, __LINE__, msg) \
-  [ (x) ? 1 : -1 ] __attr_unused___
+  [ (x) ? 1 : -1 ] __attribute__((unused))
```

Comment:
I think it looks weird, so I kept the attribute here.
Do you have a sophisticated solution? :)